scripts/wood_density.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, **kwargs):
         2. Provide recognition of the efforts of this group in the assembly of the data by using the citation for the database above.
         3. Recognize that these data were assembled by the group for various analyses and research questions. If any of these uses overlap with your interests, you recognize that group has precedence in addressing these questions."""
 
+        if VERSION < 2.1.dev:
```

Comment:
@henrykironde I looked into this for a bit. I guess we could be better off with `parse_version`.
Since `LooseVersion` can be easily tricked for some complex version use cases like - 
```
>>> LooseVersion('1.4') > LooseVersion('1.4-rc1')
False
```
With `parse_version` - 
```
>>> parse_version('1.4') > parse_version('1.4-rc2')
True
```
I know that we are not using such complex versions, but still I guess `parse_version` would be a more better choice. Kindly let me know if there are any changes required with the comparison of versions.